src/processor/marketplace-events/index.ts

```diff
@@ -24,6 +24,7 @@ import { errorHandler as originalErrorHandler } from '../../utils'
 
 const logger: Logger = loggingFactory('processor:cache')
 const NEW_BLOCK_EVENT = 'newBlock'
+export const REORG_EVENT = 'reorg-event'
```

Comment:
I would incorporate into the naming fact that the reorg is outside of the confirmation range, because the "normal reorgs" are handled by Cache and Pinner does not have to worry about those.

Reply:
Done